extensions/sql-migration/src/models/stateMachine.ts

```diff
@@ -100,6 +100,10 @@ export class MigrationStateModel implements Model, vscode.Disposable {
 	public _subscriptions!: azureResource.AzureResourceSubscription[];
 
 	public _targetSubscription!: azureResource.AzureResourceSubscription;
+	public _locations!: Location[];
```

Comment:
Generally I would expect arrays to start off initialized (especially if you're using !)